### DISCUSSION TITLE
[Script request] Opnsense

### DISCUSSION THREAD

**Discussion by jsapede** [community-scripts/ProxmoxVE#182] [Request script] [answered]
### Application Name

Opnsense

### Website

https://opnsense.org/

### Description

OPNsense is an open source, easy-to-use and easy-to-build FreeBSD based firewall and routing platform. OPNsense includes most of the features available in expensive commercial firewalls, and more in many cases. It brings the rich feature set of commercial offerings with the benefits of open and verifiable sources.

OPNsense started as a fork of pfSense® and m0n0wall in 2014, with its first official release in January 2015. The project has evolved very quickly while still retaining  familiar aspects of both m0n0wall and pfSense. A strong focus on security and code quality drives the development of the project.

OPNsense offers weekly security updates with small increments to react on new emerging threats within in a fashionable time.  A fixed release cycle of 2 major releases each year offers businesses the opportunity to plan upgrades ahead. For each major release a roadmap is put in place to guide development and set out clear goals.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by oerix** [comment]
+1 hopefully this will be added soon.

**Reply by oerix** [reply]
VM added in [latest release](https://github.com/community-scripts/ProxmoxVE/releases/tag/2025-02-11). 🎉

**Reply by tremor021** [reply, marked as answer]
Resolved with #2097
